uiowa/uiowa
# [Alerts] Current alerts block

Resolves https://github.com/uiowa/uiowa/issues/6658




# How to test

`composer install` if you haven't recently


Default
```
ddev blt ds --site=default && ddev drush @default.local config-split:activate sitenow_alerts -y && ddev drush @default.local uli
```

Facilities
```
ddev blt ds --site=facilities.uiowa.edu && ddev drush @facilities.local uli
```

- Add some events. Past, current, future. Categorize.
- Confirm custom help text from previous work is still displayed with the scheduling options and the published checkbox.
- As a webmaster, add the Alerts list block
- Note that only currently active, published alerts are displayed sorted by start time, earliest to latest (after sticky)
- Try filtering by alert categories.
- More link should work with custom path, otherwise defaults to `<front>`
- Try hiding fields.
- Review teaser styling.
